.circleci/config.yml

```diff
@@ -0,0 +1,244 @@
+
+    - persist_to_workspace:
+        root: /var/www/html
+        paths:
+          - .
+
+# Run Drupal unit and kernel tests as one job. This command invokes the test.sh
+# hook.
+unit_kernel_tests: &unit_kernel_tests
+  <<: *defaults
+  steps:
+    - attach_workspace:
+        at: /var/www/html
+
+    - checkout
+
+    - run:
+        working_directory: /var/www/html
+        command: |
+          cp ./modules/apigee_m10n/.circleci/update-dependencies.sh /var/www/html
+          cp ./modules/apigee_m10n/.circleci/RoboFile.php /var/www/html
+          ./test.sh apigee_m10n
+
+    - store_test_results:
+        path: /var/www/html/artifacts/phpunit
+    - store_artifacts:
+        path: /var/www/html/artifacts
+
+# Run Drupal functional tests. This command invokes the test-functional.sh
+# hook.
+functional_tests: &functional_tests
+  <<: *defaults
+  steps:
+    - attach_workspace:
+        at: /var/www/html
+
+    - checkout
+
+    - run:
+        working_directory: /var/www/html
+        command: |
+          cp ./modules/apigee_m10n/.circleci/update-dependencies.sh /var/www/html
+          cp ./modules/apigee_m10n/.circleci/RoboFile.php /var/www/html
+          ./test-functional.sh apigee_m10n
+
+    - store_test_results:
+        path: /var/www/html/artifacts/phpunit
+    - store_artifacts:
+        path: /var/www/html/artifacts
+
+# Run Drupal functional tests. This command invokes the test-functional-js.sh
+# hook.
+functional_js_tests: &functional_js_tests
+  <<: *defaults
+  steps:
+    - attach_workspace:
+        at: /var/www/html
+
+    - checkout
+
+    - run:
+        working_directory: /var/www/html
+        command: |
+          cp ./modules/apigee_m10n/.circleci/update-dependencies.sh /var/www/html
+          cp ./modules/apigee_m10n/.circleci/RoboFile.php /var/www/html
+          ./test-functional-js.sh apigee_m10n
+
+    - store_test_results:
+        path: /var/www/html/artifacts/phpunit
+    - store_artifacts:
+        path: /var/www/html/artifacts
+
+# Run Behat tests. This command invokes the behat.sh hook.
+behat_tests: &behat_tests
+  <<: *defaults
+  steps:
+    - attach_workspace:
+        at: /var/www/html
+
+    - checkout
+
+    - run:
+        working_directory: /var/www/html
+        command: |
+          cp ./modules/apigee_m10n/.circleci/update-dependencies.sh /var/www/html
+          cp ./modules/apigee_m10n/.circleci/RoboFile.php /var/www/html
+          ./behat.sh apigee_m10n
+
+    - store_artifacts:
+        path: /var/www/html/artifacts
+
+# Run code quality tests. This invokes code-sniffer.sh.
+code_sniffer: &code_sniffer
+  <<: *defaults
+  steps:
+    - attach_workspace:
+        at: /var/www/html
+
+    - checkout
+
+    - run:
+        working_directory: /var/www/html
+        command: |
+          cp ./modules/apigee_m10n/.circleci/update-dependencies.sh /var/www/html
+          cp ./modules/apigee_m10n/.circleci/RoboFile.php /var/www/html
+          ./code-sniffer.sh apigee_m10n
```

Comment:
I think this is using the standard drupal rule set. Should we make it use the `phpcs.xml.dist` that is included with the module?